Geometry/HcalSimData/data/v1/HcalProdCuts.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```

Comment:
Full path needs to be ` Geometry/HcalSimData/data/HcalProdCuts/2021/v2/HcalProdCuts.xml`. Make sure you derive `v2` from the `v1` file.